packages/integration/lib/src/views/search/search_screen.dart

```diff
@@ -542,6 +546,217 @@ class LMFeedSearchScreenState extends State<LMFeedSearchScreen> {
     );
   }
 
+  Widget _defPollWidget(LMFeedPoll pollWidget, LMPostViewData postViewData) {
```

Comment:
Return type should be LMFeedPollWidget

Reply:
here return type will be `Widget` since it is origially called in `LMFeedPostMedia` where the return type is `LMFeedPollWidget` on feed_screen and other screens we are just ovverriding them with it's implementation